class/cert-manager.yml

```diff
@@ -1,11 +1,42 @@
 parameters:
+  =_config:
+    # --- Exoscale Webhook
+    exoscale_webhook_values:
+      'True':
+        input_paths:
+          - ${_base_directory}/component/helm/exoscale-webhook.jsonnet
+        input_type: jsonnet
+        output_path: ${_base_directory}/helmcharts/exoscale-webhook/${cert_manager:charts:exoscale_webhook:version}/
+      'False': &option_false
+        input_paths: []
+        input_type: jsonnet
+        output_path: ''
+    exoscale_webhook_helm:
+      'True':
+        input_paths:
+          - ${_base_directory}/helmcharts/exoscale-webhook/${cert_manager:charts:exoscale_webhook:version}/
+        input_type: helm
+        helm_values_files:
+          - ${_base_directory}/helmcharts/exoscale-webhook/${cert_manager:charts:exoscale_webhook:version}/values-component.yaml
+          - ${_base_directory}/helmcharts/exoscale-webhook/${cert_manager:charts:exoscale_webhook:version}/values-overrides.yaml
+        helm_params:
+          name: exoscale-webhook
+          namespace: ${cert_manager:namespace}
+        output_path: cert-manager/20_exoscale_webhook
+      'False': *option_false
+
   kapitan:
     dependencies:
       - type: helm
         source: ${cert_manager:charts:cert_manager:source}
         chart_name: cert-manager
         version: ${cert_manager:charts:cert_manager:version}
         output_path: ${_base_directory}/helmcharts/cert-manager/${cert_manager:charts:cert_manager:version}/
+      - type: git
```

Comment:
Always downloading the dependency isn't very nice, but making it optional is not trivial given the current Kapitan feature set